docs/farming-&-staking/farming/advanced-cli/platforms/_linux.mdx

```diff
@@ -93,14 +93,20 @@ import styles from '@site/src/pages/index.module.css';
 ```bash
   # Replace `INSERT_YOUR_ID` with a nickname you choose
   # Copy all of the lines below, they are all part of the same command
-  ./subspace-node-ubuntu-x86_64-skylake-gemini-3g-2024-jan-08 \
-    --chain gemini-3g \
+  ./subspace-node-ubuntu-x86_64-skylake-gemini-3g-2024-jan-08 run \
+    --base-path path_to_node \
+    --chain gemini-3h \
     --blocks-pruning 256 \
     --state-pruning archive-canonical \
     --no-private-ipv4 \
```

Comment:
I believe this option has been removed or changed as it no longer works as a CLI parameter.

cc @nazar-pc